pages/common/vim.md

```diff
@@ -1,37 +1,37 @@
 # vim
 
-> Vi IMproved, a programmer's text editor, provides several modes for different kinds of text manipulation.
+> Vim (Vi IMproved), a programmer's command-line text editor, provides several modes for different kinds of text manipulation.
 > Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
 > More information: <https://www.vim.org>.
 
 - Open a file:
 
-`vim {{file}}`
+`vim {{path/to/file}}`
 
-- Enter text editing mode (insert mode):
+- Save a file:
 
-`<Esc>i`
+`:write<Enter>`
 
-- Copy ("yank") or cut ("delete") the current line (paste it with `P`):
+- Quit without saving:
 
-`<Esc>{{yy|dd}}`
+`:quit!<Enter>`
 
-- Undo the last operation:
+- Open a file at a specified line number:
 
-`<Esc>u`
+`vim +{{line_number}} {{path/to/file}}`
 
-- Search for a pattern in the file (press `n`/`N` to go to next/previous match):
+- View Vim's help manual:
```

Comment:
Perhaps this should be the second example?

Reply:
Great idea. I'll change that in the morning.